src/mlpack/core/optimizers/adam/optimisticadam_update.hpp

```diff
@@ -0,0 +1,144 @@
+/**
+ * @file optimisticadam_update.hpp
+ * @author Moksh Jain
+ *
+ * OptmisticAdam optimizer. Optimistic Adam is an an algorithm which uses
```

Comment:
Are you sure the algorithm's name is `OptimisticAdam` or `Optimistic Adam`? Ideally, only one should be here and at other places. Also, it seems like a typo here at `an an `.

Reply:
The name of the algorithm is `Optimistic Adam`. `OptimisticAdam` is the name of the class implementing the optimizer. Regarding the extra `an`, I'll fix that, thanks!

Reply:
You should mention `Optimistic Adam` whenever describing the algorithm. `OptimisticAdam` should be mentioned when the class or its associated subparts are concerned.